chainer/optimizer.py

```diff
@@ -690,21 +690,6 @@ def update(self, lossfun=None, *args, **kwds):
         self.call_hooks('post')
 
     def use_cleargrads(self, use=True):
-        """Enables or disables use of :func:`~chainer.Link.cleargrads` in `update`.
```

Comment:
This is a documented api and thus cannot be removed.
https://docs.chainer.org/en/stable/reference/generated/chainer.GradientMethod.html#chainer.GradientMethod